c/include/libsbp/settings.h

```diff
@@ -51,22 +50,40 @@ such strings on reads.
  * The save settings message persists the device's current settings
  * configuration to its onboard flash memory file system.
  */
-#define SBP_MSG_SETTINGS_SAVE          0x00A1
+#define SBP_MSG_SETTINGS_SAVE        0x00A1
 
 
-/** Read setting by direct index (host <=> device)
+/** Read setting by direct index (host => device)
  *
  * The settings message for iterating through the settings
  * values. It will read the setting at an index, returning a
  * NULL-terminated and delimited string with contents
- * [SECTION_SETTING, SETTING, VALUE].
+ * [SECTION_SETTING, SETTING, VALUE] in MSG_SETTINGS_READ_DEVICE.
  */
-#define SBP_MSG_SETTINGS_READ_BY_INDEX 0x00A2
+#define SBP_MSG_SETTINGS_READ_HOST   0x00A2
```

Comment:
We can pivot and change everything back over to REQUEST/RESPONSE. Seems like a better scheme - I think the concern was we didn't want to proliferate REQ/RESP pattern in piksi communication. For these cases, though, I think Request (host) / Response (device) is apt and trying to avoid a REQ/RESP pattern has instead caused more problems than its solved. (I too have found it confusing figuring out the direction of HOST/DEVICE despite coming up with the naming :).